ecosystem/sep-0009.md

```diff
@@ -109,6 +109,7 @@ applications use fields that are the most familiar, which are often specific to
 | `bank_number`         | string | Number identifying bank in national banking system (routing number in US)      |
 | `bank_phone_number`   | string | Phone number with country code for bank                                        |
 | `bank_branch_number`  | string | Number identifying bank branch                                                 |
+| `memo`                | string | A destination tag/memo used to identify a transaction                          |
```

Comment:
WDYT about renaming it to `external_transfer_memo`? 
So it's not confused with Stellar's transaction memo

Reply:
That sounds good! And what do you think about deprecation of `crypto_memo` in that case? We can use `external_transfer_memo` for both regular bank and crypto transfers, right? @Ifropc @marcinx

Reply:
Cool! It is updated.